tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -625,6 +628,32 @@ install_ovmf_sev() {
 	install_ovmf "sev" "edk2-sev.tar.gz"
 }
 
+install_agent_helper() {
+	agent_policy="${1:-no}"
+
+	latest_artefact="$(git log -1 --pretty=format:"%h" ${repo_root_dir}/src/agent)"
+	latest_builder_image="$(get_agent_image_name)"
+
+	install_cached_tarball_component \
```

Comment:
I might have missed something, but does this `agent_policy` value need to change the `install_cached_tarball_component` to differentiate between the cached agent with and without policy?

Reply:
We should just use the `build_target` as the component name, I think.